code.js

```diff
@@ -1,22 +1,18 @@
 var input = document.getElementById("input");
 var output = document.getElementById("output");
 var allInputs = JSON.parse(localStorage.getItem("allInputs")) || [];
-var setInputs = localStorage.setItem("allInputs", JSON.stringify(allInputs));
 
 if (input) {
   input.addEventListener("keypress", function (event) {
     if (event.key === "Enter") {
       event.preventDefault();
-      console.log(input.value);
       allInputs.push(input.value);
       input.value = "";
-      setInputs;
-      output.innerHTML = allInputs;
+      localStorage.setItem("allInputs", JSON.stringify(allInputs));
+      for (i = 0; i < allInputs.length; i++) {
+        console.log(allInputs[i]); // wyświetla wszystkie
+        output.innerHTML = allInputs[i]; // tylko ostatni
+      }
```

Comment:
W tym miejscu wywołujemy funkcję do renderowania zadań

Reply:
1. Tworzysz sobie nową pozycję w divie np. Robisz to za poomocą pętli map.
<div>
{nazwa_zadania}
</div>